upgrade/1.2/Stage_0_Prerequisites.md

```diff
@@ -2,50 +2,56 @@
 
 > **NOTE:** CSM-1.0.1 or higher is required in order to upgrade to CSM-1.2.0
 
-* [Stage 0.1 - Install latest docs RPM](#install-latest-docs)
-* [Stage 0.2 - Reduce CPU Limits If Needed](#reduce-cpu-limits)
-* [Stage 0.3 - Update SLS](#update-sls)
-* [Stage 0.4 - Upgrade Management Network](#update-management-network)
-* [Stage 0.5 - Prerequisites Check](#prerequisites-check)
-* [Stage 0.6 - Backup VCS Data](#backup-vcs-data)
-* [Stage 0.7 - Suspend NCN Configuration](#suspend-ncn-config)
-* [Stage 0.8 - Backup Workload Manager Data](#backup_workload_manager)
-* [Stage 0.9 - Modify NCN Images](#modify_ncn_images)
-* [Stage 0.10 - Continue to Stage 1](#continue_to_stage1)
+- [Stage 0.1 - Install latest docs RPM](#install-latest-docs)
+- [Stage 0.2 - Reduce CPU Limits If Needed](#reduce-cpu-limits)
+- [Stage 0.3 - Update SLS](#update-sls)
+- [Stage 0.4 - Upgrade Management Network](#update-management-network)
+- [Stage 0.5 - Prerequisites Check](#prerequisites-check)
+- [Stage 0.6 - Backup Workload Manager Data](#backup_workload_manager)
+- [Stage 0.7 - Continue to Stage 1](#continue_to_stage1)
 
 <a name="install-latest-docs"></a>
+
 ## Stage 0.1 - Install latest docs RPM
 
-1. Install latest document RPM package:
+1. Install latest document RPM package and prepare assets:
+
+   > The install scripts will look for the RPM in `/root`, so it is important that you copy it there.
 
-    > The install scripts will look for the RPM in `/root`, so it is important that you copy it there.
+   - Internet Connected
 
-    * Internet Connected
+     ```bash
```

Comment:
These commands are missing the "ncn-m001" prompts.

Unless we are going to change our documentation conventions, we should still be following the command prompt conventions. We cannot just be inconsistent about it and have it down to the preference of the person doing the PR.

Making a sweeping change like that is beyond the scope of this PR, so for now we should follow the stated conventions. If you want to change those conventions, I have no objection, but that should be its own ticket.